pkg/cli/store.go

```diff
@@ -37,7 +37,7 @@ func init() {
 
 	set.Int("csub-listen-port", 2782, "port to listen to on collect-sub service")
 
-	set.String("gql-backend", "inmem", "backend used for graphql api server: [neo4j | inmem]")
+	set.String("gql-backend", "inmem", "backend used for graphql api server: [neo4j | inmem | arango]")
```

Comment:
let's reorder this to inmem in front and add qualifiers, like: `inmem | arango (experimental) | neo4j (unmaintained)`

Reply:
added.